CMakeLists.txt

```diff
@@ -115,6 +115,13 @@ else()
     set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE NEVER)
 
     # Add our own target properties
+    # General module definitions
+    define_property(TARGET PROPERTY REACTOS_MODULE_TYPE
+        BRIEF_DOCS "The type of this module"
+        FULL_DOCS [[
+he type of this module.
+One of "nativecui", "nativedll", "kernelmodedriver", "wdmdriver", "kerneldll", "win32cui", "win32gui", "win32dll", "win32ocx", "cpl" or "module"]])
```

Comment:
Can you tell me (or refer) what "module" (in that list) means? i.e. What sort of subsystem/default libs/.... values it describes?

Reply:
Just that : a DLL which is not meant to have a particular entry point, nor anything fancy